tests/acceptance/signin-test.js

```diff
@@ -1,4 +1,3 @@
-import deparam from 'npm:deparam';
```

Comment:
The two places this was used have been removed so we can fully remove this dependency.